packages/mask/src/plugins/Tips/components/TipButton.tsx

```diff
@@ -94,6 +100,27 @@ export const TipButton: FC<Props> = ({
         return wallets
     }, [receiver])
 
+    const walletsFromCloud = useMemo(() => {
+        if (kv?.ok) {
+            if (!kv.val.proofs.length) return null
+            const tipWallets = kv.val.proofs.map((x) =>
+                x.content[PluginId.Tips].filter((y) => y.platform === NextIDPlatform.Ethereum),
+            )[0]
+            if (!tipWallets) return []
+            return tipWallets
+                .filter((x) => {
+                    if (NextIDWalletsState.value) {
+                        // Sometimes, the wallet might get deleted from next.id
+                        return x.isPublic && NextIDWalletsState.value.includes(x.identity)
+                    } else {
+                        return x.isPublic
+                    }
+                })
+                .map((x) => x.identity)
+        }
+        return null
```

Comment:
```suggestion
        if (kv?.ok) {
            if (!kv.val.proofs.length) return EMPTY_LIST
            const tipWallets = kv.val.proofs.map((x) =>
                x.content[PluginId.Tips].filter((y) => y.platform === NextIDPlatform.Ethereum),
            )[0]
            if (!tipWallets) return EMPTY_LIST
            return tipWallets
                .filter((x) => {
                    if (NextIDWalletsState.value) {
                        // Sometimes, the wallet might get deleted from next.id
                        return x.isPublic && NextIDWalletsState.value.includes(x.identity)
                    } else {
                        return x.isPublic
                    }
                })
                .map((x) => x.identity)
        }
        return EMPTY_LIST
```

Reply:
User might not do any settings, then walletsFromCloud would be `null`
`    const hideAllWallets = walletsFromCloud !== null && walletsFromCloud.length === 0`